en/sim_sih/index.md

```diff
@@ -164,6 +164,40 @@ export PX4_HOME_ALT=28.5
 make px4_sitl sihsim_quadx
 ```
 
+
+### Adding new airframes
+
+When adding new airframes for use in SIH simulation, there are some differences with respect to ususal airframe files. 
+
+For all variants of SIH:
+
+  - Set the [physical model parameters](../advanced_config/parameter_reference.md#simulation-in-hardware).
+  - `param set-default SYS_HITL 2` to enable SIH on the next boot.
```

Comment:
Comments in airframe says this also stops sensors.

I'm not quite sure how this works w.r.t. sensors. It looks like in the SIH on SITL the sensors must be enabled such as SENS_EN_GPSSIM 1` . On FC I assume that the simulator must take care of providing simulated data for some subset of sensors. Isn't clear if these can be enabled/disabled in the usual way.